doc/releases/changelog-0.9.0.md

```diff
@@ -166,204 +180,221 @@
+  purposes.
   [(#1179)](https://github.com/PennyLaneAI/catalyst/pull/1179)
 
-* Scalar tensors are eliminated from control flow operations in the program, and are replaced with
-  bare scalars instead. This improves compilation time and memory usage at runtime by avoiding heap
-  allocations and reducing the amount of instructions.
-  [(#1075)](https://github.com/PennyLaneAI/catalyst/pull/1075)
+  ```python
+  qml.device("null.qubit", wires=1)
 
-  A new MLIR pass `detensorize-scf` is added that works in conjuction with the existing
-  `linalg-detensorize` pass to detensorize input programs. The IR generated by JAX wraps all values
-  in the program in tensors, including scalars, leading to unnecessary memory allocations for
-  programs compiled to CPU via MLIR to LLVM pipeline.
+  @qml.qjit
+  @qml.qnode(dev)
+  def g(x):
+      qml.RX(x, wires=0)
+      return qml.probs(wires=[0])
+  ```
 
-* Bufferization of `gradient.ForwardOp` and `gradient.ReverseOp` now requires 3 steps:
-  `gradient-preprocessing`, `gradient-bufferize`, and `gradient-postprocessing`.
-  `gradient-bufferize` has a new rewrite for `gradient.ReturnOp`.
-  [(#1139)](https://github.com/PennyLaneAI/catalyst/pull/1139)
+* Setting the `seed` argument in the `qjit` decorator will now seed sampled results, in addition to
+  mid-circuit measurement results.
+  [(#1164)](https://github.com/PennyLaneAI/catalyst/pull/1164)
 
-* The decorator `self_inverses` now supports all Hermitian Gates.
-  [(#1136)](https://github.com/PennyLaneAI/catalyst/pull/1136)
+  ```python
+  dev = qml.device("lightning.qubit", wires=1, shots=10)
+
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(x, wires=0)
+      m = catalyst.measure(0)
+
+      if m:
+          qml.Hadamard(0)
+
+      return qml.sample()
+
+  @qml.qjit(seed=37, autograph=True)
+  def workflow(x):
+      return jnp.squeeze(jnp.stack([circuit(x) for i in range(4)]))
+  ```
 
-  The full list of supported gates are as follows:
+  ```pycon
+  >>> workflow(1.8)
+  Array([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [1, 1, 0, 0, 1, 1, 0, 0, 1, 0],
+         [0, 0, 1, 0, 1, 1, 0, 0, 1, 1],
+         [1, 1, 1, 0, 0, 1, 1, 0, 1, 1]], dtype=int64)
+  >>> workflow(1.8)
+  Array([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [1, 1, 0, 0, 1, 1, 0, 0, 1, 0],
+         [0, 0, 1, 0, 1, 1, 0, 0, 1, 1],
+         [1, 1, 1, 0, 0, 1, 1, 0, 1, 1]], dtype=int64)
+  ```
 
-  One-bit Gates:
-  - [`qml.Hadamard`](https://docs.pennylane.ai/en/stable/code/api/pennylane.Hadamard.html)
-  - [`qml.PauliX`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliX.html)
-  - [`qml.PauliY`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliY.html)
-  - [`qml.PauliZ`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliZ.html)
+  Note that statistical measurement processes such as `expval`, `var`, and `probs` are currently not
+  affected by seeding when shot noise is present.
```

Comment:
From Josh (https://github.com/PennyLaneAI/catalyst/pull/1239#discussion_r1820392616):

> @paul0403 this caught me off-guard/feels like a bit of a bug. I was expecting expval(...) to also be seeded in the presence of shot noise, but it looks like it still fluctuates?

Reply:
Resolved in #1248.